docs/applications/development/clojure-deployment-with-immutant-and-wildfly-on-ubuntu-14-04.md

```diff
@@ -231,9 +231,9 @@ To deploy Clojure application with WildFly you will need to install Immutant lei
 	}
 	~~~
 
-	This sets the destination folder to copy the WAR file into, context path and WAR file name - which should be ROOT for root context path.
+	This sets the destination folder into which the WAR file, context path and WAR file name are copied - which should be ROOT for root context path.
```

Comment:
Wording here is a bit incorrect, as I see it - original meaning is that three parameters will be set - destination folder for WAR file, context path and WAR file name. Looks like I made it unclear, but "to copy into" only relates to destination folder, not to context path and WAR file name.